### ISSUE TITLE
Unable to find status bar in Office aplications

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#649]
**Reported by ManuelRamos on 2010-05-13 23:56**
It will be very usefull that NVDA can report status line in Microsoft Word, Excel, and so on. 
**Blocked by #3323**

**Comment by leonardder** [comment]
I can read both the status bar in Word and Excel 2016 using nvda+shift+end in laptop lay-out

**Comment by Adriani90** [comment]
Testing on MS Word 2016 and MS Excel 2016: Zoom factor, Word count and macro status etc is reported when pressing NVDA+end or NVDA+shift+end. So at least in word and Excel 2016 I cannot reproduce it. I will test tomorrow in Office 2010.

**Comment by josephsl** [comment]
Hu, How about in 2013/ If someone can confirm this is working in 2013 (at least in majority of cases), then I vote this issue be closed. Thanks.

**Comment by Adriani90** [comment]
@fernando-jose-silva, could you please test this with Office 2013? Thank you.

**Comment by fernando-jose-silva** [comment]
@Adriani90
I believe I was not the person I wanted to mention, I had not said anything about this call, but I'm glad to be able to help.
I did the test in word 2013 and nvda read the status bar correctly.
Thank you very much.

**Comment by ehollig** [comment]
Thank you @fernando-jose-silva for testing in Word 2013. Closing, as suggested in https://github.com/nvaccess/nvda/issues/649#issuecomment-404828752
